### ISSUE TITLE
One Liners Appearing as Name and Not as Dialogue

### ISSUE THREAD

**Issue by whimsicaljoey** [Beamdog/nwn-issues#560]
- Load up original campaign.
 - Full text bubble mode is on.
 - Speak to any Citizen in City Core, when clicked for dialogue it only shows their name.
 - When Doomsayer says his text, it only shows up as his name.

**Specifics**

 - OS (Windows, Mac, Linux) - Windows
 - Game build (e.g. 8193.8, or commit hash) - 8193.36
 - Distro platform (BDC, Steam, GOG) - Steam
 - Content overrides? (Steam Workshop? Overrides? Patch haks?) - None
 - Regression? (Build which broke it, if known) - N/A

**If needed, describe the bug**

Basically, one liners only show up in the chat below and not as floating text over NPCs heads.

**Comment by Finaldeath** [comment]
![image](https://github.com/Beamdog/nwn-issues/assets/6137144/c7d3c8d8-71b2-4691-ba7c-e3ae681f9627)

Verified. Odd! If you can test an old version (on Steam you can select them in the builds menu) please do to check if it's a kind of regression from past EE patches.

**Comment by whimsicaljoey** [comment]
> ![image](https://private-user-images.githubusercontent.com/6137144/293437430-c7d3c8d8-71b2-4691-ba7c-e3ae681f9627.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDM4OTgzNDEsIm5iZiI6MTcwMzg5ODA0MSwicGF0aCI6Ii82MTM3MTQ0LzI5MzQzNzQzMC1jN2QzYzhkOC03MWIyLTQ2OTEtYmE3Yy1lM2FlNjgxZjk2MjcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDIzMTIzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMzBUMDEwMDQxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2Y1YjIyYWU4NTY5M2I3ODFhYzQ4MmI0N2I3ZGRjN2RmZDFkZTgzZWJlYTg2ZDA3MDc2ODY3ZGY1OGRkMmZhOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.BPxTI1idHVzf9y7zkPMXMZ3wlIfQmWnjuBqM_gM_Rls)
> 
> (1.69 above)
> 
> Verified. Odd! If you can test an old version (on Steam you can select them in the builds menu) please do to check if it's a kind of regression from past EE patches.

![35 sc 1](https://github.com/Beamdog/nwn-issues/assets/77301034/76b5e986-d5a8-4c04-a38d-92ec6fcc7a33)
![36 sc 1](https://github.com/Beamdog/nwn-issues/assets/77301034/776f0d02-d35b-4e00-9712-8a9a893099d5)

This issue is only present on 8193.36. I also tested it with entering a trigger that has one line of dialogue and it did not work. The Nurse and Tomi Undergallows' one liners (Hey! Do you need a henchman?) work fine. This issue is not at all present in 8193.35.

**Comment by Finaldeath** [comment]
Ok a regression on 36, thanks for reporting hopefully we'll fix it before stable.

**Comment by Finaldeath** [comment]
Was fixed a while back - cleaning up.
